src/csvcubed/models/cube/qb/components/codelist.py

```diff
@@ -175,22 +212,8 @@ class CompositeQbCodeList(NewQbCodeList[DuplicatedQbConcept]):
 
     variant_of_uris: List[str] = field(default_factory=list)
 
-
-def validate_codelist(
-    item: QbCodeList, property_name: str
-) -> List[ValidateModelProperiesError]:
-    if not isinstance(item, QbCodeList):
-        return [
-            ValidateModelProperiesError(
-                f"This variable should be a QbCodeList, check the following variable:",
-                property_name,
-            )
-        ]
-
-    """ 
-    TODO: when the class is inctanciated for the validations the function has to be called.
-    example: 
-    test = Myclass(argument1, argument2, argument3)
-    test.validate()
-    """
-    return []
+    def _get_validations(self) -> Dict[str, ValidationFunction]:
+        return {
+            **NewQbCodeList._get_validations(self),
+            "variant_of_uris": validate_list(validate_str_type),
```

Comment:
This should be validating URIs inside the list.

Reply:
Done